actions/signed-commits/dist/index.js

```diff
@@ -60142,15 +60147,13 @@ async function getLocalTags(cwd) {
   });
   return await Promise.all(tags);
 }
-async function createLightweightTags(tagSeparator, tags, cwd) {
+async function createLightweightTags(tags, cwd) {
   const createdTags = tags.map(async (tag) => {
-    const newTagName = tag.name.replace("@", tagSeparator);
-    await execWithOutput("git", ["tag", newTagName, tag.ref], { cwd });
-    return {
-      name: newTagName,
-      ref: tag.ref,
-      originalName: newTagName != tag.name ? tag.name : void 0
-    };
+    const maybeForceFlag = tag.majorVersion ? ["-f"] : [];
+    await execWithOutput("git", ["tag", tag.name, tag.ref, ...maybeForceFlag], {
```

Comment:
Are we signing the tags?

Reply:
The tags are lightweight so they point to signed commits.

Annotated tags would require signing, and actually the whole reason we have this fork is because `changesets` makes annotated tags by default, and this fork rewrote them to lightweight tags.